src/main/java/org/opensearch/security/tools/AuditConfigMigrater.java

```diff
@@ -43,6 +45,8 @@ public class AuditConfigMigrater {
     private static final HelpFormatter formatter = new HelpFormatter();
     private static final CommandLineParser parser = new DefaultParser();
 
+    static final Logger log = LogManager.getLogger(AuditConfigMigrater.class);
```

Comment:
There aren't any tests on this tool, this could be introducing breaking changes in the log output, what do you think about avoiding this file?

Reply:
Same for Mitgrater.java